internal/pkg/v2/handler/core/data/event.go

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package data
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/correlation"
+
+	"github.com/edgexfoundry/edgex-go/internal/core/data/config"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/interfaces"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+	model "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+
+	"github.com/edgexfoundry/go-mod-messaging/messaging"
+	msgTypes "github.com/edgexfoundry/go-mod-messaging/pkg/types"
+)
+
+func AddNewEvent(
```

Comment:
rename to AddEvent, and add some simple go doc

Reply:
Updated.